modules/rosa-setting-up-cli.adoc

```diff
@@ -0,0 +1,46 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-get-started-cli.adoc
+
+
+[id="rosa-setting-up-cli_{context}"]
+= Setting up the `rosa` CLI
```

Comment:
As I think Jeana mentioned, we decided against using backticks in a heading.

Reply:
Done.